### PR TITLE
Text changes in README

### PR DESCRIPTION
**Pull request by mistergone**
selfie-0

* links to Contributing.md
* changed "should consider writing" to "should write"
* changed "Extend" to "How to Use"
* Changed warning to be more direct. (User doesn't really need to know _why_ they have to use PHP 5.3+.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,12 +40,12 @@ To activate the plugin follow the steps below:
 
 ## Develop
 
-See Contributing.md
+See [Contributing.md](Contributing.md)
```
Comment (kurtw) [inline README.md]: Link is case sensitive. "Contributing" needs to be all uppercase.

**Comment by kurtw** [comment]
@mistergone hey chuck, I know you're busy but could you attend to the note when you get a chance? Thanks.

**Comment by mistergone** [comment]
Sorry, that should do it!

**Comment by kurtw** [comment]
Looks great! Thanks.

**Pull request merged**